### PR TITLE
[lldb][NFC] Add a test for big multi payload enums

### PR DESCRIPTION
**Pull request by augusto2112**

### CODE REVIEW

**Comment by augusto2112** [comment]
@swift-ci test

File: lldb/test/API/lang/swift/big_mpe/TestSwiftBigMPE.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+
+
+class TestSwiftBigMPE(TestBase):
```
Comment (adrian-prantl) [inline lldb/test/API/lang/swift/big_mpe/TestSwiftBigMPE.py]: would you mind spelling out MPE here and in the directory name? Might make it easier to grep for this later.

File: lldb/test/API/lang/swift/big_mpe/TestSwiftBigMPE.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    @swiftTest
+    def test(self):
+        self.build()
+        lldbutil.run_to_source_breakpoint(
+            self, "break here", lldb.SBFileSpec("main.swift")
+        )
+
+        self.expect("frame variable a1", substrs=["a.Request", "a1 = a1"])
```
Comment (adrian-prantl) [inline lldb/test/API/lang/swift/big_mpe/TestSwiftBigMPE.py]: any advandage if we use lldbutil.check_variable() here?

**Comment by augusto2112** [comment]
@swift-ci test

**Review by adrian-prantl — APPROVED** [review]

**Comment by augusto2112** [comment]
@swift-ci test Linux

**Pull request merged**